pyro/contrib/tracking/hashing.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import, division, print_function
+
+import itertools
+from collections import defaultdict
+from numbers import Number
+
+
+class LSH(object):
+    """
+    Implements locality-sensitive hash for low-dimensional euclidean space.
+    """
+    def __init__(self, scale):
```

Comment:
Can we rename `scale` to `radius` and describe the radius guarantee in the docstring?